rubocop-airbnb/rubocop-airbnb.gemspec

```diff
@@ -25,9 +25,9 @@ Gem::Specification.new do |spec|
     'Gemfile',
   ]
 
-  spec.add_dependency('rubocop', '~> 1.32.0')
-  spec.add_dependency('rubocop-performance', '~> 1.10.2')
-  spec.add_dependency('rubocop-rails', '~> 2.9.1')
-  spec.add_dependency('rubocop-rspec', '~> 2.0.0')
+  spec.add_dependency('rubocop', '>= 1.61.0')
```

Comment:
Use `~>` instead of `>=` to keep the dependency on the same major version. Also, if Ruby 3.3 support was added in `1.46.0`, would it be better for this to be `~> 1.46.0` instead?

Reply:
Yeah, `~>` seems better.
With `rubocop-performance ~> 1.20.2` (which is the first version that tests against `ruby 3.3`) we will need at least `rubocop 1.48.1`, the thing with that version is that when running tests, it results in this error: 

```
➜  rubocop-airbnb git:(gm--support_ruby_3.3) ✗ be rspec

An error occurred while loading spec_helper.
Failure/Error: require 'rubocop-rspec'

RuboCop::ValidationError:
  Property AutoCorrect of cop FactoryBot/CreateList is supposed to be a boolean and contextual is not.
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_validator.rb:261:in `block in check_cop_config_value'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_validator.rb:254:in `each'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_validator.rb:254:in `check_cop_config_value'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_validator.rb:255:in `block in check_cop_config_value'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_validator.rb:254:in `check_cop_config_value'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_validator.rb:34:in `validate'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config.rb:55:in `check'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config.rb:26:in `create'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_loader.rb:57:in `load_file'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-1.48.1/lib/rubocop/config_loader.rb:150:in `inject_defaults!'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-factory_bot-2.26.0/lib/rubocop-factory_bot.rb:16:in `<top (required)>'
# ./vendor/bundle/ruby/3.3.0+0/gems/rubocop-rspec-2.26.1/lib/rubocop-rspec.rb:8:in `<top (required)>'
# ./lib/rubocop/cop/airbnb/spec_constant_assignment.rb:1:in `<top (required)>'
# ./lib/rubocop/airbnb.rb:4:in `map'
# ./lib/rubocop/airbnb.rb:4:in `<top (required)>'
# ./lib/rubocop-airbnb.rb:10:in `<top (required)>'
# ./spec/spec_helper.rb:10:in `<top (required)>'
```

That stops happening from version, 1.61.0 of rubocop, I mention that in [this commit's message](https://github.com/airbnb/ruby/pull/205/commits/95b7a6fe15f776f9c80019676b5ed87ca56c9576)
Also see: https://github.com/rubocop/rubocop/blob/45ba7790b3c6fc37c0b83693513c4dadd619d1ed/CHANGELOG.md?plain=1#L19
[This](https://github.com/rubocop/rubocop/pull/12657) is the PR that adds the marked change in the previous link. Note that 'AutoCorrect: disabled maintains compatibility with AutoCorrect: false.' so I don't think it would be necessary replacing the `false` values.